src/Firebase/Messaging/MessageData.php

```diff
@@ -51,9 +51,9 @@ public function jsonSerialize(): array
     /**
      * @param mixed $value
      */
-    private static function isStringable($value): bool
+    private static function isStringableAndNumeric($value): bool
```

Comment:
The method would have to check if the value is numeric to make sense: with a value of `"string"`, the method would still return `true`